src/afrolabs/config.clj

```diff
@@ -75,9 +77,22 @@
 (defmethod aero/reader 'regex
   [_ _ value] (re-pattern value))
 
+(defn interpret-string-as-csv-row
+  [row-string]
+  (first (csv/read-csv (java.io.StringReader. row-string))))
+
 (defmethod aero/reader 'csv-array
   [_ _ value]
-  (first (csv/read-csv (java.io.StringReader. value))))
+  (interpret-string-as-csv-row value))
+
+(defmethod aero/reader 'long?
+  [_ _ value]
+  (when (and value
+             (seq value))
+    (Long/parseLong (str value))))
```

Comment:
new aero (config file) reader tag for values when if present, must become long values, otherwise being nillable.